scenes/inventory/item.gd

```diff
@@ -0,0 +1,12 @@
+class_name Item
+extends Resource
+
+enum ItemType {
+	MEMORY,
+	IMAGINATION,
+	SPIRIT
```

Comment:
Although this is not a requirement of current linter, I suggest a trailing comma here. Explanation: https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#trailing-comma
```suggestion
	SPIRIT,
```